tests/ci/ci_config.py

```diff
@@ -828,9 +826,6 @@ def validate(self) -> None:
         ),
     },
     other_jobs_configs={
-        JobNames.MARK_RELEASE_READY: TestConfig(
```

Comment:
It just removes the logic for `Ready for release` from ci.py, and we will control it through the workflow yml files, right?

Reply:
yeap